grow/pods/rendered.py

```diff
@@ -49,18 +50,33 @@ def render(self, params, inject=True):
             preprocessor = self.pod.inject_preprocessors(doc=self.doc)
             translator = self.pod.inject_translators(doc=self.doc)
         env = self.pod.get_jinja_env(self.locale)
+
         local_tags = tags.create_builtin_tags(
             self.pod, self.doc, use_cache=self.pod.env.cached)
         template = env.get_template(self.view.lstrip('/'))
         # NOTE: This should be done using get_template(... globals=...) but
         # it is not available included inside macros???
         # See: https://github.com/pallets/jinja/issues/688
         template.globals['g'] = local_tags
+
+        # Configure the footnotes based on the doc or podspec settings.
+        footnote_config = self.doc.fields.get(
+            '$footnotes', self.pod.podspec.fields.get('footnotes', {}))
+        locale = str(self.doc.locale) if self.doc.locale else None
+        symbols = footnote_config.get('symbols', None)
+        use_numeric_symbols = footnote_config.get('use_numeric_symbols', None)
+        numeric_locales_pattern = footnote_config.get(
+            'numeric_locales_pattern', None)
+        notes = footnotes.Footnotes(
+            locale, symbols=symbols, use_numeric_symbols=use_numeric_symbols,
+            numeric_locales_pattern=numeric_locales_pattern)
+
         try:
             kwargs = {
                 'doc': self.doc,
                 'env': self.pod.env,
-                'podspec': self.pod.get_podspec(),
+                'footnotes': notes,
```

Comment:
I think this should probably be a property on the `Document` object to avoid overloading our global with extra built-in vars. The footnotes are also document-specific so I think that would make sense.